.idea/scopes/JavaInspectionsScope.xml

```diff
@@ -1,3 +1,3 @@
 <component name="DependencyValidationManager">
-  <scope name="JavaInspectionsScope" pattern="file:*.java||file:pom.xml" />
+  <scope name="JavaInspectionsScope" pattern="file:*.java||file:pom.xml||lib:*..*" />
```

Comment:
why this change?

Reply:
Pushed local change by mistake. Thanks for pointing, will revert it.